### ISSUE TITLE
groups: viewing certain group messages crashes browser tab

### ISSUE THREAD

**Issue by alecananian** [tloncorp/landscape-apps#2340]
**Describe the bug, optionally with screenshots**
Certain Groups message views crash my browser tab upon opening while others are fine (e.g. Tlon > Watercooler crashes but not Tlon > Homestead). Apps are installed off wannec, and I can reproduce it locally on the `master` branch as well. No console output or concerning Network activity is shown.

**To Reproduce**
1. Open Groups or Talk
2. Click on Tlon > Watercooler channel
3. Browser tab freezes

**Expected behavior**
All channels are accessible

**Additional context:**
- Tlon-hosted planet
- macOS 13.3.1 Chromium and Android Firefox

**Comment by urcades** [comment]
A video capturing this behavior in action:

https://user-images.githubusercontent.com/1195363/233159057-fd6b5ff1-1253-41f4-910d-8ca65cec62a2.mov

Up until about 26 seconds, I'm able to load Talk successfully, click into a channel I've recently been active in, and load chat history without issue. After this, I click into a group channel I host, attempt to scroll back, and get caught up in the observed issue with pages becoming unresponsive.

**Comment by joemfb** [comment]
I'm also seeing this, across several channels. LMK if the details would be helpful

**Comment by lukestiles** [comment]
> 

Yes please @joemfb - all insight welcome. Also Mark and Hunter are pairing in Gather. If you have time to join please do.

**Comment by Fang-** [comment]
Indeed also seeing this, specifically in my DM with @joemfb. (So, maybe don't DM me? Unclear if that would destroy the repro.)

We're in the Tlon Gather, but can hop over elsewhere if there's useful new deets. Currently trying to look at a browser memory snapshot that is taking hilariously long to save.

**Comment by Fang-** [comment]
@arthyn and @patosullivan identified the bug and found the fix! Patch soon, I imagine.

**Comment by joemfb** [comment]
Just got out of a meeting, but looks like this is handled. I'll be around if it turns out that more info is needed.
